looper/cli_pydantic.py

```diff
@@ -167,16 +241,112 @@ def run_looper(args: TopLevelParser, parser: ArgumentParser):
                 )
                 raise
 
+        if subcommand_name in ["runp"]:
+            compute_kwargs = _proc_resources_spec(subcommand_args)
+            collate = Collator(prj)
+            collate(subcommand_args, **compute_kwargs)
+            return collate.debug
+
+        if subcommand_name in ["destroy"]:
+            return Destroyer(prj)(subcommand_args)
 
-def main() -> None:
-    parser = pydantic_argparse.ArgumentParser(
+        use_pipestat = (
+            prj.pipestat_configured_project
+            if getattr(subcommand_args, "project", None)
+            else prj.pipestat_configured
+        )
+
+        if subcommand_name in ["table"]:
```

Comment:
why are you using `in` here instead of `==`? You have a list of one variable.

Reply:
fixed